site/scripts/create-db-indexes.ts

```diff
@@ -0,0 +1,53 @@
+import { User, UserDocument } from "../src/lib/model/user.model";
+import {
+  VerificationCode,
+  VerificationCodeDocument,
+} from "../src/lib/model/verificationCode.model";
+import { connectToDatabase } from "../src/lib/mongodb";
+import { EntityType } from "../src/lib/model/entityType.model";
+
+const catchAndLog = async (func: () => Promise<void>) => {
```

Comment:
It might be useful to add a small comment here explaining why we catch and log instead of thowing and terminating the whole script.